distribution/docs/src/main/resources/_contents/_broker/route-manager-contents.adoc

```diff
@@ -0,0 +1,21 @@
+==== Route Manager
+
```

Comment:
It's a good idea for safety to start these files with a blank space so your header doesn't get clobbered by whatever comes before it.